db/seeds/development/rooms_seed.rb

```diff
@@ -0,0 +1,8 @@
+conference = Conference.first
+(1..25).each do |index|
+  new_room = Room.create(
+    conference_id: conference.id,
```

Comment:
You can just do `conference: conference` and rails will magically assign the `id` correctly!